### PR TITLE
minimalize initial doc and move it to libvirt

### PR DESCRIPTION
**Pull request by MalloZup**
move the libvirt doc to libvirt. 
Make the initial doc a index only

# you can see it here:
https://github.com/MalloZup/ha-sap-terraform-deployments/tree/doc-readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,20 @@
...
-For more informations have a look at [pillar-doc](pillar_examples/README.md)
+- [libvirt/KVM](libvirt)
+- [azure](azure)
+- [google cloud](gcp)
+- [aws](aws)
 
 ___
 ## Rationale:
 
 This project is organized in folders containing the Terraform configuration files per Public or Private Cloud providers, each also containing documentation relevant to the use of the configuration files and to the cloud provider itself.
```
Comment (ayoub-belarbi) [inline README.md]: Do we have or intend to have a private cloud provider support in this project ?

Comment (MalloZup) [reply README.md]: I think yes we should maintain what we have or remove them

File: README.md [hunk 2]
```diff
@@ -1,38 +1,20 @@
 # Automated SAP/HA Deployments in Public/Private Clouds with Terraform
 
 [![Build Status](https://travis-ci.org/SUSE/ha-sap-terraform-deployments.svg?branch=master)](https://travis-ci.org/SUSE/ha-sap-terraform-deployments)
-Terraform Version supported `0.12.6`
+**Terraform Version supported `0.12.6`**
 ___
-# Quickstart:
 
-In this quickstart `libvirt` it is used  as example but same logic can be applied to other terraform providers.
+# Cloud providers supported:
```
Comment (arbulu89) [inline README.md]: This is quite strange. Shouldn't be?:
```Supported cloud providers```

File: README.md [hunk 3]
```diff
@@ -1,38 +1,20 @@
 # Automated SAP/HA Deployments in Public/Private Clouds with Terraform
 
 [![Build Status](https://travis-ci.org/SUSE/ha-sap-terraform-deployments.svg?branch=master)](https://travis-ci.org/SUSE/ha-sap-terraform-deployments)
-Terraform Version supported `0.12.6`
+**Terraform Version supported `0.12.6`**
```
Comment (arbulu89) [inline README.md]: Shouldn't be `Supported terraform version` ?

File: README.md [hunk 4]
```diff
@@ -1,38 +1,20 @@
...
-
-3) Adapt pillars:
-
-  Choose one profile, among the list. (in this example we choose `cost_optimized`)
-
-  * from root top-level dir:
-   `cp pillar_examples/libvirt/cost_optimized/*  salt/hana_node/files/pillar`
-
-For more informations have a look at [pillar-doc](pillar_examples/README.md)
+- [libvirt/KVM](libvirt)
```
Comment (arbulu89) [inline README.md]: It's not a cloud provider, but it's OK anyway

File: README.md [hunk 5]
```diff
@@ -1,38 +1,20 @@
...
 
 This project is organized in folders containing the Terraform configuration files per Public or Private Cloud providers, each also containing documentation relevant to the use of the configuration files and to the cloud provider itself.
 
-The documentation of terraform and the cloud providers included in this repository is not intended to be complete, so be sure to also check the [documentation provided by terraform](https://www.terraform.io/docs) and the cloud providers.
-
-___
-## Terraform version
-
-All Terraform configurations were tested with the 0.12.6 version
+This project uses Terraform for deploy and Saltstack for provisioning
```
Comment (arbulu89) [inline README.md]: I would rewrite as:
`This project uses Terraform for the deployment and Saltstack for the provisioning`

File: libvirt/README.md [hunk 6]
```diff
@@ -19,90 +19,27 @@
 
 2. You need to fulfill the system requirements provided by SAP for each Application. At least 15 GB of free disk space and 512 MiB of free memory per node.
 
-# Howto
+# Quickstart
 
-To deploy the cluster only the parameters of three files should be changed:
+In this quickstart `libvirt` it is used  as example but same logic can be applied to other terraform providers.
```
Comment (arbulu89) [inline libvirt/README.md]: The `it` after libvirt is not needed.
Double space after `used`.
I would recheck the whole sentence actually

File: libvirt/README.md [hunk 7]
```diff
@@ -19,90 +19,27 @@
...
-#}
-# To disable the provisioning process
-#provisioner = ""
-```
-
-After changing the values, run the terraform commands:
-
-```bash
-terraform workspace new myworkspace # The workspace name will be used to create the name of the created resources as prefix (`default` by default)
-terraform init
```
Comment (arbulu89) [inline libvirt/README.md]: Maybe we should keep the `init`, `apply` and `destroy` options.

File: README.md (hunk 2 above)
Comment (MalloZup) [reply README.md]: done thx

File: README.md (hunk 3 above)
Comment (MalloZup) [reply README.md]: done thx

File: README.md (hunk 5 above)
Comment (MalloZup) [reply README.md]: done thx

File: libvirt/README.md (hunk 6 above)
Comment (MalloZup) [reply libvirt/README.md]: done

File: libvirt/README.md (hunk 7 above)
Comment (MalloZup) [reply libvirt/README.md]: done thx

Comment (MalloZup) [reply libvirt/README.md]: done thx

File: libvirt/README.md [hunk 8]
```diff
@@ -19,90 +19,34 @@
...
-```
-
-After changing the values, run the terraform commands:
+4) Deploy with:
 
 ```bash
 terraform workspace new myworkspace # The workspace name will be used to create the name of the created resources as prefix (`default` by default)
 terraform init
 terraform apply
+terraform destroy
```
Comment (arbulu89) [inline libvirt/README.md]: Maybe we should add a comment before the destroy (it looks obvious, what who knows hehe)

Comment (MalloZup) [reply libvirt/README.md]: I think it is ok to have it without comment. I mean I suppose it is really self event what destroy mean and also the terraform destroy command warn you that you are going to destroy.

We can't fix the problem betweeen the chair and the computer :grin:

Comment (MalloZup) [reply libvirt/README.md]: I think is fine without the comment. I would not be against if you add it.. But I think it is not needed .. thx for review

**Pull request merged**